pkg/operator/ceph/csi/csi.go

```diff
@@ -84,6 +84,14 @@ func SetParams(clientset kubernetes.Interface) error {
 		return errors.Wrap(err, "unable to parse value for 'ROOK_CSI_ENABLE_GRPC_METRICS'")
 	}
 
+	csiEnableCSIHostNetwork, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_ENABLE_HOST_NETWORK", "false")
+	if err != nil {
+		return errors.Wrap(err, "unable to determine if CSI Host Network is enabled")
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to determine if CSI Host Network is enabled")
```

Rook is using `failed to` across its code base to produce error message.